lib/transformations/utils.js

```diff
@@ -83,7 +83,7 @@ function findPluginsRootNodes(j, node) {
 function createProperty(j, key, value) {
 	return j.property(
 		'init',
-		j.literal(key),
+		j.identifier(key),
```

Comment:
Yeah, so key in a Property can be Identifier or Literal, so for safety we'll keep it Literal?